internal/impl/laravel/5.8/handler/generator/model_generator.go

```diff
@@ -0,0 +1,146 @@
+Returns:
+	- instance of the generator object
+*/
+func (modelGenerator *ModelGenerator) AddUpdateValidationRule(colName string, colRule string) *ModelGenerator {
+	if modelGenerator.updateValidationRules == nil {
+		modelGenerator.updateValidationRules = make(map[string]string)
+	}
+	modelGenerator.updateValidationRules[colName] = colRule
+	return modelGenerator
+}
+
+/**
+ Set's the name of the model class.
+ Parameters:
+	- tableName: the name of table specified in asher config.
+ Returns:
+	- instance of the generator object
+ */
+func (modelGenerator *ModelGenerator) SetName(tableName string) *ModelGenerator {
+	className := strcase.ToCamel(tableName)
+	modelGenerator.classBuilder.SetName(className)
+	return modelGenerator
+}
+
+/**
+ Adds the given column to fillable fields of this model
+ Parameters:
+	- columnName: the name of the column to add
+ Returns:
+	- instance of the generator object
+ */
+func (modelGenerator *ModelGenerator) AddFillable(columnName string) *ModelGenerator {
+	modelGenerator.fillables = append(modelGenerator.fillables, `"` + columnName + `"`)
+	return modelGenerator
+}
+
+/**
+ Adds the given column to hidden fields of this model
+ Parameters:
+	- columnName: the name of the column to add
+ Returns:
+	- instance of the generator object
+*/
+func (modelGenerator *ModelGenerator) AddHiddenField(columnName string) *ModelGenerator {
+	modelGenerator.hidden = append(modelGenerator.hidden, `"` + columnName + `"`)
+	return modelGenerator
+}
+
+/**
+ Control whether to set timestamps in the model of not
+ Returns:
+	- instance of the generator object
+*/
+func (modelGenerator *ModelGenerator) SetTimestamps(flag bool) *ModelGenerator {
+	modelGenerator.timestamps = flag
+	return modelGenerator
+}
+
+func (modelGenerator *ModelGenerator) Build() *core.Class {
+	modelGenerator.classBuilder = modelGenerator.classBuilder.SetPackage("App").AddImports([]string{
+		`Illuminate\Database\Eloquent\Model`,
+	}).SetExtends("Model")
+
+	if len(modelGenerator.fillables) > 0 {
+		fillableArray := api.TabbedUnit(core.NewArrayAssignment("protected", "fillable",
+			modelGenerator.fillables))
+		modelGenerator.classBuilder = modelGenerator.classBuilder.AddMember(&fillableArray)
+	}
+
+	if len(modelGenerator.hidden) > 0 {
+		hiddenArray := api.TabbedUnit(core.NewArrayAssignment("protected", "visible",
+			modelGenerator.hidden))
+		modelGenerator.classBuilder = modelGenerator.classBuilder.AddMember(&hiddenArray)
+	}
+
+	if modelGenerator.timestamps {
+		timestamps := api.TabbedUnit(core.NewVarAssignment("public","timestamps", "true"))
+		modelGenerator.classBuilder = modelGenerator.classBuilder.AddMember(&timestamps)
+	}
+
+	if len(modelGenerator.createValidationRules) > 0 {
+		returnArray := api.TabbedUnit( core.NewReturnArrayFromMap(modelGenerator.createValidationRules) )
+		createFunction := builder.NewFunctionBuilder().SetName("createValidationRules").
+			SetVisibility("public").AddStatement(&returnArray).GetFunction()
```

Comment:
change to getCreateValidationRules() and make it static
Do this for update also

Reply:
Done